### PR TITLE
EPMLSTRCMW-272 Create postman collection for RunController

### PR DESCRIPTION
**Pull request by Nikolai-Isaev**
docs: Add Runs postman collection

### CODE REVIEW

**Review by DruzhininPavel — COMMENTED** [review]
LGTM

**Review by DruzhininPavel — APPROVED** [review]

**Review by gominasian — APPROVED** [review]

**Comment by myazinn** [comment]
Collection in this PR doesn't correspond to current API scheme. Please mark your PR as draft so that we don't merge it accidentally.

**Comment by Nikolai-Isaev** [comment]
Oops, sorry about that. Drafted

**Comment by myazinn** [comment]
Now this PR is ready for review, right?

**Comment by Nikolai-Isaev** [comment]
> Now this PR is ready for review, right?

Yep, marked it "ready for review" again

**Review by myazinn — APPROVED** [review]

File: postman/Runs.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+						"projects",
+						"{{projectId}}",
+						"runs"
+					]
+				}
+			},
+			"response": []
+		}
+	]
+}
```
Comment (iliaonishchenko) [inline postman/Runs.postman_collection.json]: Please add new line at the end of the file :)

**Review by iliaonishchenko — APPROVED** [review]
It looks good to me.

File: postman/Runs.postman_collection.json (hunk 1 above)
Comment (Nikolai-Isaev) [reply postman/Runs.postman_collection.json]: Done

**Comment by Nikolai-Isaev** [comment]
Readme.md renewed

**Pull request merged**